roles/config-rh-sso/templates/realm_roles.json.j2

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "{{ role_data.role_id | default(omit) }}",
+  "name": "{{ role_data.role_name }}",
+  "description": "{{ role_data.role_description | default('') }}",
+  "attributes": "{{ role_data.role_attributes | default(omit) }}",
+  "clientRole": "{{ role_data.role_clientRole | default(false) }}",
+  "composite":  "{{ role_data.role_composite | default(false) }}",
+  "composites": "{{ role_data.role_composites | default(omit) }}",
+  "containerId": "{{ role_data.role_containerId | default('master') }}"
```

Comment:
What does  containerId refer to? Is this referring to the realm?

Reply:
Yes. What I observed during the JSON fetch, is that a manually created Realm level role returns the container ID with the name of the Role it belongs to, however, to the API, it doesn't seem to matter much, as this value is marked as optional.

Reply:
So name of the role? Or name of the realm? Just want to be careful with this because it looks like we're defaulting it to master (and want to make sure that's an appropriate value).

Reply:
The `containerId` expects either the Realm name or the client name so that the role can me associated with. This field is marked as optional but I defaulted it to the `Master Realm` here whether no values are deinfed, we can try to modify that if we need or adjust this value.

Reply:
Let's go ahead and default it to the realm value